sql/src/main/java/io/crate/execution/engine/join/HashInnerJoinBatchIterator.java

```diff
@@ -110,18 +106,15 @@ public HashInnerJoinBatchIterator(RamAccountingBatchIterator<L> left,
                                       CircuitBreaker cicuitBreaker,
                                       long estimatedRowSizeForLeft,
                                       long numberOfRowsForLeft,
-                                      int limit,
-                                      boolean isOrdered) {
+                                      int rowsToBeConsumed) {
```

Comment:
pls pass in the BlockSizeCalculator as all 3 vars are only interested to this component